examples/transloadit/index.html

```diff
@@ -29,7 +29,7 @@
     <main>
       <h1>Robodog <img src="https://transloadit.edgly.net/assets/images/logo-small.svg" alt="Transloadit" id="logo"> playground</h1>
       <p>
-        This page contains small examples for every API offered by the Robodog library. Please see the <a href="https://github.com/transloadit/uppy/tree/master/examples/transloadit">Github repository</a> for the source code.
+        This page contains small examples for every API offered by the Robodog library. Please see the <a href="https://github.com/transloadit/uppy/tree/main/examples/transloadit">Github repository</a> for the source code.
```

Comment:
Should we use `HEAD` instead of `main` in case we change the default branch name again (however unlikely it is)?
```suggestion
        This page contains small examples for every API offered by the Robodog library. Please see the <a href="https://github.com/transloadit/uppy/tree/HEAD/examples/transloadit">Github repository</a> for the source code.
```

Reply:
I'm no git expert, so I don't know if there are any drawbacks of using HEAD instead of main

Reply:
@goto-bus-stop @kvz `HEAD` vs `main`?

Reply:
I think we have a lot of places where we reference `main` now so only changing it here wouldn't help much if we change the branch name again. We can leave it as is in my opinion.

Reply:
> I think we have a lot of places where we reference `main` now so only changing it here wouldn't help much if we change the branch name again.

I think the idea if someone was copying the link to somewhere else (StackOverflow, a documentation, etc.), they would copy `HEAD` instead of `main`, and the link would not break if we ever change the default branch name again. It's still a very low probability anyway, so I don't feel strongly about it.